constructor/main.py

```diff
@@ -230,14 +228,14 @@ def main_build(
                 "Will assume it is compatible with shortcuts."
             )
         elif sys.platform != "win32" and (
-            exe_type != StandaloneExe.CONDA or (exe_version and exe_version < Version("23.11.0"))
+            exe_type != StandaloneExe.CONDA or not check_version(exe_version, min_version="23.11.0")
         ):
             logger.warning("conda-standalone 23.11.0 or above is required for shortcuts on Unix.")
             info["_enable_shortcuts"] = "incompatible"
 
     # Add --no-rc option to CONDA_EXE command so that existing
     # .condarc files do not pollute the installation process.
-    if exe_type == StandaloneExe.CONDA and exe_version and exe_version >= Version("24.9.0"):
+    if exe_type == StandaloneExe.CONDA and check_version(exe_version, min_version="24.9.0"):
```

Comment:
I know this may be a little out of scope but there's no logger warning or comment regarding the need to use `24.9.0` here

Reply:
Yes, it's out of scope. It's not really an opt-in feature but a bug fix. If `conda` didn't crash when using an unknown flag, none of these checks would even be necessary. A logger comment would be necessary if this was something a `constructor` user controlled, but couldn't have.